docs/provider-kubernetes.md

```diff
@@ -92,4 +85,69 @@ spec:
     remoteRef:
       key: secret-example
       property: extra
+```
+
+### Remote Secret using a Token
+
+1. Create a K8s Secret with the encoded base64 ca and client token.
+   
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cluster-secrets
+data:
+  # Fill with your encoded base64 CA
+  certificate-authority-data: Cg==
+stringData:
+  # Fill with your string Token
+  bearerToken: "my-token"
+```
+2. Create a SecretStore
+
+The Server section specifies the `url` of the remote Kubernetes API. In this example the Certificate Authority is fetch using the encoded base64 `caBundle`. 
+
+The `auth` section indicates that the  `token` type will be used for authentication, it includes the path to fetch the token.
+
+```
+apiVersion: external-secrets.io/v1beta1
+kind: SecretStore
+metadata:
+  name: example
+spec:
+  provider:
+      kubernetes: 
+        # If not remoteNamesapce is provided, default     namespace is used
+        remoteNamespace: remote-namespace
+        server: 
+          url:  https://127.0.0.1:36473
```

Comment:
I think we can use a dns proxy, like `https://remote.kubernetes.api-server.address`